subtests/docker_cli/run_sigproxy/run_sigproxy.py

```diff
@@ -98,7 +112,7 @@ def initialize(self):
             self._init_container_attached(name)
         else:
             self._init_container_normal(name)
-        time.sleep(self.config['wait_start'])
+        self._wait_for_ready()
```

Comment:
>what about adding it to the output module?

Good idea, so done, tested, and then:

   (pylint) C0302:  1,0: : Too many lines in module (1020/1000)

*whimper*